MANIFEST.in

```diff
@@ -2,3 +2,4 @@ recursive-include evals *.py
 recursive-include evals *.yaml
 recursive-include evals *.sql
 recursive-include evals/registry/data *.jsonl
+recursive-include evals *.jsonl
```

Comment:
We probably don't need this, can just keep the previous line: `recursive-include evals/registry/data *.jsonl`